backend/zeno_backend/classes/homepage.py

```diff
@@ -0,0 +1,21 @@
+"""Classes for homepage data."""
+
+from zeno_backend.classes.base import CamelModel
+from zeno_backend.classes.project import Project
+from zeno_backend.classes.report import Report
+
+
+class HomepageData(CamelModel):
+    """Projects, reports, and metadata for homepage rendering.
+
+    Attributes:
+        projects (list[Project]): the projects to be displayed.
+        reports (list[Report]): the reports to be displayed.
+        num_projects (int): the number of projects.
+        num_reports (int): the number of reports.
+    """
+
+    projects: list[Project]
+    reports: list[Report]
+    num_projects: int
+    num_reports: int
```

Comment:
I don't think I would fetch this as one. We're saving one request but will have to refactor big time if we ever change/disentangle this again.

Reply:
So four different requests?

Reply:
IMO worth not making as many sequential fetch requests we await, makes it significantly slower

Reply:
I would do two requests and they can be parallel. One for reports, one for projects. I'm pretty sure there will be cases where we only fetch one.